src/build

```diff
@@ -17,9 +17,6 @@ if [[ $platform == 'linux' ]]; then
     if hash qmake-qt5 2>/dev/null; then
 	echo "Using qmake-qt5"
 	QCMD=qmake-qt5
-    elif hash qmake-qt4 2>/dev/null; then
```

Comment:
this check for `qmake-qt4` was causing a compile failure if `qmake` defaults to Qt5 but `qmake-qt4` is also present. Since Qt4 is apparently no longer supported, this seemed fine to remove.